templates/primary-secondary/primary-secondary.js

```diff
@@ -591,7 +590,8 @@ class TemplatePrimarySecondary extends RtlMixin(LocalizeCoreElement(LitElement))
 			_isCollapsed: { type: Boolean, attribute: false },
 			_isExpanded: { type: Boolean, attribute: false },
 			_isMobile: { type: Boolean, attribute: false },
-			_size: { type: Number, attribute: false }
+			_size: { type: Number, attribute: false },
+			_sizeAsPercent: { state: true }
```

Comment:
I needed to make this a reactive state property so that when it gets changed when the window resizes it triggers a re-render.